src/plugins/codeStyle/index.ts

```diff
@@ -170,20 +170,20 @@ export default class CodeStyle {
         file.addDiagnostics(bsDiagnostics);
     }
 
-    validateColorFormat(fileContents: string, diagnostics: (Omit<BsDiagnostic, 'file'>)[], colorFormat: RuleColorFormat) {
+    validateColorFormat(token: AALiteralExpression, diagnostics: (Omit<BsDiagnostic, 'file'>)[], colorFormat: RuleColorFormat) {
```

Comment:
This should be `LiteralExpression`, not `AALiteralExpression`. 

LiteralExpressions are things like `"hello"`, `"1.23"`.

AALiteralExpressions are things like `[1, 2, 3]`